src/utils/makeBranch.js

```diff
@@ -22,7 +23,9 @@ module.exports = async function(api, repo, branchName) {
     cd ${repoFolder} &&
     git checkout -b ${branchName}
   `);
-  if (branchCreate.stderr) {
-    console.warn(' - error creating branch', branchCreate.stderr);
-  }
+  assert(
+    branchCreate.code === 0,
+    ` - unable to create branch: ${branchCreate.stderr}`
+  );
+  console.log(' -', branchCreate.stderr);
```

Comment:
The "Switched to branch..." message is in _stderr_, so this was warning every time.